shopperz/api/views.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+from django.http import JsonResponse
+from .models import User
+from .serializers import UserSerializer
+from rest_framework.views import APIView
+
+# Create your views here.
+
+class AllViews(APIView):
```

Comment:
it would be nice to use comments to give a concise description of what this model does, for example:

```
def days_until_launch(current_day, launch_day):
    """"Returns the days left before launch.
    
    current_day (int) - current day in integer
    launch_day (int) - launch day in integer
    """
    days_left=launch_day - current_day
    
    if days_left > 0:
        return days_left
    return 0
```